sklearn/datasets/samples_generator.py

```diff
@@ -291,9 +291,10 @@ def make_multilabel_classification(n_samples=100, n_features=20, n_classes=5,
     sparse : bool, optional (default=False)
         If ``True``, return a sparse feature matrix
 
-    return_indicator : bool, optional (default=False),
-        If ``True``, return ``Y`` in the binary indicator format, else
-        return a tuple of lists of labels.
+    return_indicator : False | 'dense' | 'sparse' (default=False),
+        If ``dense`` return ``Y`` in the dense binary indicator format. If
```

Comment:
It describes what is returned, and should currently say array, not sparse matrix.